app/src/main/java/com/rentminder/service/HouseholdService.kt

```diff
@@ -12,8 +12,8 @@ class HouseholdService {
     suspend fun fetchHouseholds() : List<Household>?{
         return withContext(Dispatchers.IO) {
             val service = RetrofitClientInstance.retrofitInstance?.create(IHouseholdDAO::class.java)
-            val country = async { service?.getAllCountries() }
-            return@withContext country.await()?.awaitResponse<ArrayList<Household>>()?.body()
+            val household = async { service?.getAllHouseholds() }
+            return@withContext household.await()?.awaitResponse<ArrayList<Household>>()?.body()
```

Comment:
Use better naming conventions. Obviously this is referencing the country assignment earlier but definitely change the variable names to fit your own application so that future developers are not confused by the use of country variable. 
URL: https://kotlinlang.org/docs/coding-conventions.html#choose-good-names

Reply:
Good.